tests/conftest.py

```diff
@@ -36,3 +37,16 @@ def api_manager() -> ApiManager:
     api_manager_.reset()
     yield api_manager_
     api_manager_.__dict__.update(old_attrs)
+
+
+@pytest.fixture
+def vcr_config():
```

Comment:
Can you write a comment about how this works for those of us not familiar.  I typically expect fixtures to be invoked some where or to have `@pytest.fixture(autouse=True)`